RebelAllianceBank/Classes/SavingsAccount.cs

```diff
@@ -3,11 +3,14 @@ namespace RebelAllianceBank.Classes
 {
     public class SavingsAccount : IBankAccount
     {
+        public int ID { get; set; }
+        public string UserId { get; set; }
+        public int AccountType { get; set; }
```

Comment:
Why is it int and not string

Reply:
If you meant the UserId, it's because of it's a reference to the IUser PersonalNum